src/cargo/core/compiler/custom_build.rs

```diff
@@ -742,16 +744,18 @@ impl BuildOutput {
                 Ok(line) => line.trim(),
                 Err(..) => continue,
             };
+            let mut new_syntax = false;
             let (key, value) = if let Some(data) = line.strip_prefix("cargo::") {
+                new_syntax = true;
```

Comment:
nit: imo the "new syntax" is just the syntax.  I'd lean towards an "old syntax"

Reply:
Updated.